### ISSUE TITLE
Best of LMMS is "All rights reserved"

### ISSUE THREAD

**Issue by bkeys** [LMMS/lmms.io#191]
why is the Best of LMMS all rights reserved on BandCamp? I would think that this would go against the spirit of LMMS. The licenses for these works should be a free culture license of some sort so that more people can feel the benefits of this music (and by extension, LMMS).

**Comment by tresf** [comment]
That appears to be something bandcamp adds to the bottom of each album.  I can't find a place on the album page where we can change it.

I find it to be correct though.  Simply because a GPL product sponsors/promotes the artists' work gives LMMS no right to claim ownership or dictate licensing unless we set that as entrance criteria for the competitions (which we [didn't](https://lmms.io/competitions/).)

If you feel it's too restrictive, we'll have to work with Bandcamp to remove the text from our page, but it would have to be done moving forward (as well as written in our Vol IV rules, etc).

**Comment by Umcaruje** [comment]
Well, I know we can change the license on bandcamp (not entirely sure where, but I did it for my own releases), but I'm not sure if we can change the license now, without an explicit permission from every artist.

Though I think that having the album under a free culture license would be a good thing.

**Comment by tresf** [comment]
Per http://forum.cockos.com/showthread.php?t=33353, "All rights reserved" is no longer needed for creative works and it does not conflict with our distribution mechanism.

I'll quote the relevant parts... 

> CC license can be seen as "some rights reserved". So that means you give away some rights, e.g. the right to duplicate and share your work or the right to create derived works or only for non-commercial usage or ... or some combination of those. E.g. your stated combination: share-alike, non-derives will NOT work, since share-alike allows for sharing any derived works under the same license.
>
> Anyway Read: http://en.wikipedia.org/wiki/Creative_Commons_licenses for what type of licenses you can choose. And http://en.wikipedia.org/wiki/Creative_Commons for a more general info about CC.
>
> So CC is really a license, the copyright still persists. It's just a form of dictating how others may exercise your copyright rights. (That's why you can NOT assign a CC license to work that is not yours.)
> 
> Also you can revoke the CC license anytime you want _from the file_. Note that you can NOT remove the license granted to from those that acquired the files under the CC license. So you could then start selling it, or what ever or relicense also for commercial use etc ... just remember people that acquired the files under the original CC license can still use them according to that license.

Each artist is allowed to use whatever license they choose for BoL album (so as it doesn't conflict with our distribution mechanism) and it's not LMMS' place to tell them otherwise.
 
The only right we make them wave is to allow us to redistribute their work on our album, which is according to our own guidelines.

See, BoL is a showcase, not a free-software objective.  Free software can have serious implications on artists when it imposes itself onto works of art.  It's not in the project's best interest to micro-manage that.

On the contrary, our bundled project can and should be more open, [ideally CC0](https://github.com/LMMS/lmms/issues/1273#issuecomment-64087510), but that's a topic for a [different repo](https://github.com/LMMS/samples).

So in summary, "All rights reserved" will be there this year too.  Thanks for the attention to detail.  Closing as invalid.
